src/test/java/org/candlepin/pki/certs/AnonymousCertificateGeneratorTest.java

```diff
@@ -92,13 +94,21 @@ class AnonymousCertificateGeneratorTest {
     void setUp() throws CertificateException, IOException {
         this.config = TestConfig.defaults();
         this.config.setProperty(ConfigProperties.STANDALONE, "false");
+
+        this.generator = createGenerator();
```

Comment:
nit: fully qualify the call to createGenerator since it's not static.

Reply:
Updated